### DISCUSSION TITLE
does the merge function merge automatically the stoichiometric matrix of the two models?

### DISCUSSION THREAD

**Discussion by Ilhem36** [opencobra/cobrapy#1261] [Q&A] [answered]
Hello ! 
I have an important question please . I want to  merge two models  using thee merge function but I would like to know if the merge merge automatically the two stoichiometric matrix ? 
And how can I please acess to the stoichiometric matrix of a model with cobrapy ? 
Thank you for your answer !

**Comment by cdiener** [comment, marked as answer]
It does. You can get the stoichiometric matrix with 

```python
from cobra.util import create_stoichiometric_matrix

S = create_stoichiometric_matrix(model)
```
